thehive-charts/thehive/values.yaml

```diff
@@ -181,90 +200,116 @@ thehive:
     #      name: "cm-name"
     #      key: "key-name"
 
-  # Startup probe (set "failureThreshold" to a high value to prevent CrashLoops during schema migrations)
   startupProbe:
+    # -- Enable startup probe (set failureThreshold high to prevent CrashLoops during schema migrations)
     enabled: true
+    # -- Initial delay before startup probe begins
     initialDelaySeconds: 0
+    # -- Frequency of startup probe checks
     periodSeconds: 5
+    # -- Timeout for each startup probe attempt
     timeoutSeconds: 1
+    # -- Number of failures before marking pod as failed
     failureThreshold: 36
 
-  # Liveness probe
   livenessProbe:
+    # -- Enable liveness probe
     enabled: true
+    # -- Initial delay before liveness probe begins
     initialDelaySeconds: 0
+    # -- Frequency of liveness probe checks
     periodSeconds: 10
+    # -- Timeout for each liveness probe attempt
     timeoutSeconds: 1
+    # -- Number of failures before restarting container
     failureThreshold: 3
 
-  # Readiness probe
   readinessProbe:
+    # -- Enable readiness probe
     enabled: true
+    # -- Initial delay before readiness probe begins
     initialDelaySeconds: 0
+    # -- Frequency of readiness probe checks
     periodSeconds: 10
+    # -- Timeout for each readiness probe attempt
     timeoutSeconds: 1
+    # -- Number of successes before marking pod as ready
     successThreshold: 1
+    # -- Number of failures before marking pod as not ready
     failureThreshold: 3
 
   serviceAccount:
+    # -- Create a service account for TheHive
     create: true
+    # -- Name of the service account (generated if empty)
     name: ""
+    # -- Annotations for the service account
     annotations: {}
-    # Whether to automatically mount Kubernetes API credentials
+    # -- Automatically mount Kubernetes API credentials
     automountServiceAccountToken: true
 
-  # Pod wide security context https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+  # -- Pod-wide security context
   podSecurityContext: {}
     #fsGroup: 2000
 
-  # Container wide security context
+  # -- Container-level security context
   securityContext: {}
     #allowPrivilegeEscalation: false
 
   service:
+    # -- Kubernetes service type
     type: ClusterIP
+    # -- TheHive application port
     port: 9000
 
   monitoring:
-    # Whether to expose kamon port in TheHive Deployment and add port in TheHive Service
-    # NOTE - edit application.conf to disable kamon, see https://docs.strangebee.com/thehive/operations/monitoring/
+    # -- Enable monitoring port exposure (edit application.conf to configure Kamon)
```

Comment:
Should we specify that enabling this creates a k8s service in addition to exposing the given port on the pod side?